ddl/external_imagerec_prod.hql

```diff
@@ -20,7 +20,9 @@ CREATE EXTERNAL TABLE IF NOT EXISTS `imagerec_prod`(
   `is_article_page` boolean,
   `dataset_id` string,
   `insertion_ts` double, 
-  `found_on` array<string>)
+  `found_on` array<string>,
+  `page_namespace` int,
```

Comment:
This script was used to generate data for the API. Do we need `page_namespace` and `recommendation_type` for that use case?

Reply:
This is from a previous commit. I ended up coming to the same conclusion and just adding these columns to just the search table